static/js/controllers/reports.js

```diff
@@ -194,14 +282,17 @@ var _ = require('underscore'),
             $scope.error = false;
             $scope.errorSyntax = false;
             _updateLiveList(data);
```

Comment:
instead of using `syncCheckboxes` below, I'd add the true/false value in the `data` here, and pass it to `_updateLiveList`, which would set the right values directly.